Zend/zend_vm_def.h

```diff
@@ -2863,9 +2843,6 @@ ZEND_VM_HANDLER(47, ZEND_JMPNZ_EX, CONST|TMPVAR|CV, JMP_ADDR)
 		ZVAL_FALSE(EX_VAR(opline->result.var));
 		opline++;
 	}
-	if (UNEXPECTED(EG(exception) != NULL)) {
-		HANDLE_EXCEPTION();
-	}
```

Comment:
That's correct, I've just reviewed all EG(exception) usages and noticed these were redundant.